ts/features/bonus/bpd/components/BaseDailyTransactionHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import { View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import I18n from "../../../../i18n";
+import { H3 } from "../../../../components/core/typography/H3";
+import { H5 } from "../../../../components/core/typography/H5";
+import IconFont from "../../../../components/ui/IconFont";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+
+type Props = {
+  date: string;
+  transactionsNumber: number;
+};
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "row",
+    alignItems: "center"
+  },
+  container: {
+    justifyContent: "space-between"
+  }
+});
+
+const BaseDailyTransactionHeader: React.FunctionComponent<Props> = (
+  props: Props
+) => (
+  <>
+    <H3 weight={"Bold"} color={"black"}>
+      {props.date}
+    </H3>
+    <View style={[styles.row, styles.container]}>
+      <View style={styles.row}>
+        <IconFont
+          name={"io-transactions"}
+          size={12}
+          color={IOColors.bluegrey}
+        />
+        <H5 color={"bluegrey"}>
+          {` ${I18n.t(
+            "bonus.bpd.details.components.transactionsCountOverview.label",
+            {
+              defaultValue:
+                "bonus.bpd.details.components.transactionsCountOverview.label.other",
```

Comment:
The `defaultValue` is the fallback of type `string`.
It uses that value as a replacement, not as a translation key

this is what happens when the first key isn't found
![Schermata 2020-11-11 alle 17 45 16](https://user-images.githubusercontent.com/822471/98839576-fc598280-2445-11eb-8dbc-0b29c07bd6bd.png)

Maybe this could be fine
```typescript
{` ${I18n.t(
            "bonus.bpd.details.components.transactionsCountOverview.label2",
            {
              defaultValue: I18n.t(
                "bonus.bpd.details.components.transactionsCountOverview.label.other",
                { count: props.transactionsNumber }
              ),
              count: props.transactionsNumber
            }
          )}`}
```

Reply:
You're right i forgot the fallback was the result and not the **TranslationKey**, i'll add to next commit